config/setupTests.js

```diff
@@ -1,7 +1,7 @@
 import { configure, mount, render, shallow } from 'enzyme';
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import React from 'react';
-import 'babel-polyfill';
+import 'core-js';
```

Comment:
You might not even need `core-js`. Try running the tests without any polyfills.

Reply:
There is still missing `flatMap`, I can remove it after this project is migrated to latest Node.